src/libs/weather.ts

```diff
@@ -48,88 +21,107 @@ export class WeatherAPI {
 
 }
 
-/**
- * API Parameters and Responses
- */
-
-export interface GetWeatherResponse {
+export interface WeatherResponse {
```

Comment:
Can you keep the name? It's part of the general convention we have across the SDK.